examples/src/main/scala/org/apache/spark/examples/ml/CorrelationExample.scala

```diff
@@ -51,10 +51,10 @@ object CorrelationExample {
 
     val df = data.map(Tuple1.apply).toDF("features")
     val Row(coeff1: Matrix) = Correlation.corr(df, "features").head
-    println("Pearson correlation matrix:\n" + coeff1.toString)
+    println(s"Pearson correlation matrix:\n ${coeff1.toString}")
```

Comment:
Another thing we could improve: `.toString` is redundant here I believe

Reply:
Addressed.